api/OfflineAudioContext.json

```diff
@@ -134,10 +134,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "53"
+              "version_added": "25"
```

Comment:
Yep, the previous data didn't make any sense this the constructor is the entry point to this API.